scripts/build_release.sh

```diff
@@ -102,91 +91,96 @@ make build
 
 scripts/build_packages.sh "${PLATFORM}"
 
-# Run RPM bulid in Centos7 Docker container
-sg docker "docker build -t algocentosbuild - < scripts/centos-build.Dockerfile"
-
-# cleanup our libsodium build
-if [ -f ${GOPATH}/src/github.com/algorand/go-algorand/crypto/libsodium-fork/Makefile ]; then
-    (cd ${GOPATH}/src/github.com/algorand/go-algorand/crypto/libsodium-fork && make distclean)
-fi
-rm -rf ${GOPATH}/src/github.com/algorand/go-algorand/crypto/lib
 
-# do the RPM build
-sg docker "docker run --env-file ${HOME}/build_env_docker --mount type=bind,src=${GOPATH}/src,dst=/root/go/src --mount type=bind,src=${HOME},dst=/root/subhome --mount type=bind,src=/usr/local/go,dst=/usr/local/go -a stdout -a stderr algocentosbuild /root/go/src/github.com/algorand/go-algorand/scripts/build_release_centos_docker.sh"
+# Test .deb installer
 
-# Tag Source
+mkdir -p ${HOME}/docker_test_resources
```

Comment:
I don't want these tests to be part of the build - we need to do more comprehensive tests of the deb and rpm; the build should just generate the binaries / packages and not also do tests.
Please extract the testing process to separate script(s) that can be run manually against the pushed but not promoted packages.